spec/jobs/ingest_file_job_spec.rb

```diff
@@ -5,11 +5,17 @@
   let(:filename) { fixture_file_path('/world.png') }
   let(:user)     { create(:user) }
 
-  it 'uses the provided mime_type' do
+  it 'uses a provided mime_type' do
     described_class.perform_now(file_set.id, filename, 'image/png', 'bob')
     expect(file_set.reload.original_file.mime_type).to eq 'image/png'
   end
 
+  it 'does not decorate File when not given mime_type' do
+    allow(CurationConcerns::VersioningService).to receive(:create)
```

Comment:
Why should VersioningService ever be called? wasn't `versioning: false` going to prevent it?